src/main/java/com/exadel/discountwebapp/discount/controller/DiscountController.java

```diff
@@ -48,4 +51,16 @@ public DiscountResponseVO update(@PathVariable Long id, @Valid @RequestBody Disc
     public void deleteById(@PathVariable Long id) {
         discountService.deleteById(id);
     }
-}
+
+    @PostMapping("/favorites")
+    public ResponseEntity<String> addDiscountToFavorites(@RequestParam(value = "userId") Long userId, @RequestParam(value = "discountId") Long discountId) {
```

Comment:
ResponseEntity<String>
does this method return the response body as a String?

Reply:
changed to void